dot/parachain/network-bridge/receiver.go

```diff
@@ -300,134 +220,12 @@ func (nbr *NetworkBridgeReceiver) updateOurView() error {
 	// TODO: send ViewUpdate to all the collation peers and validation peers (v1, v2, v3)
 	// https://github.com/paritytech/polkadot-sdk/blob/aa68ea58f389c2aa4eefab4bf7bc7b787dd56580/polkadot/node/network/bridge/src/rx/mod.rs#L969-L1013
 
-	// TODO: Create our view and send collation events to all subsystems about our view change
+	// TODO #4156 Create our view and send collation events to all subsystems about our view change
```

Comment:
I thought this PR addresses issue #4156?  Should a new issue be created, or is the more to add to this PR?

Reply:
I am actually gonna keep #4156 open. Merge this PR with `kishan/feat/network-bridge-rx`. Create one more PR where I can close #4156